client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/sidebar.tsx

```diff
@@ -23,9 +27,13 @@ const Sidebar = ( { siteSlug }: { siteSlug: string | null } ) => {
 					</div>
 				</div>
 				{ /* eslint-disable-next-line wpcalypso/jsx-classname-namespace*/ }
-				<h1 className="launchpad__sidebar-h1">Voilà! Your Newsletter is up and running!</h1>
+				<h1 className="launchpad__sidebar-h1">
+					{ translate( 'Voilà! Your %(translatedFlowName)s is up and running!', {
```

Comment:
Im not sure, but maybe worth separating the "Voila!" out into its own translation. My hunch is that is slightly more difficult to translate consistently, and if we end up translating the full sentence in the switch case like my other comment it might be less pressure on translators and more consistent if we leave Voila! on its own.

Reply:
Because we're translating whole strings (based on comment above), we determined this is not needed.